src/Notion/Properties/Select.php

```diff
@@ -11,6 +11,10 @@ public function value()
 
     public function set($value): void
     {
+        if (!isset($this->config->select)) {
+            $this->config->select = new \stdClass();
```

Comment:
I'd prefer `(object) []` instead of the `new \stdClass();`

Reply:
Should be easy to resolve like this
```suggestion
            $this->config->select = (object) [];
```